examples/crowdfunding/assets/crowdFundApproval.teal

```diff
@@ -83,11 +83,11 @@ txn Sender
 ==
 
 // check that we are past fund close date
-// global LatestTimestamp
-// byte "FundCloseDate"
-// app_global_get
-// >=
-// &&
+global LatestTimestamp
```

Comment:
Please merge master(this will be updated)

Reply:
This was discussed on call, as a task @man-of-code will be reviewing this PR, so i haven't merged master yet (intentionally).